webrender_api/src/api.rs

```diff
@@ -714,10 +714,21 @@ impl RenderApiSender {
             channel::msg_channel().expect("Failed to create channel");
         let msg = ApiMsg::CloneApi(sync_tx);
         self.api_sender.send(msg).expect("Failed to send CloneApi message");
+        let namespace_id = match sync_rx.recv() {
```

Comment:
Maybe add a comment about how this code is for debugging an issue, so we can discard it in the future if we want.